.travis.yml

```diff
@@ -11,9 +11,9 @@ git:
 jobs:
   include:
   - env: CMD="make validate"
-  - env: CMD="make deploy deploytool=helm globalnet=true"
-  - env: CMD="make deploy deploytool=operator" DEPLOY=true
-  - env: CMD="make deploy cable_driver=wireguard"
+  - env: CMD="make deploy" CLUSTERS_ARGS="--globalnet true" DEPLOY_ARGS="--globalnet true --deploytool helm"
+  - env: CMD="make deploy" DEPLOY_ARGS="--cable_driver wireguard"
+  - env: CMD="make e2e" DEPLOY=true
```

Comment:
This should really be a separate commit, but I don’t mind either way.

Reply:
Yeah you're right if you want i can break it out

Reply:
Overall I think it would be better split out, if only to make it clearer when looking back at the history (why did we add `DEPLOY_ARGS` to Travis? no this doesn’t have anything to do with sharing e2e...).

Reply:
OK broke it out to PR #117

Reply:
Yeah I was confused by that too, thanks.